src/main/java/fixBean/BaseFixBean.java

```diff
@@ -0,0 +1,127 @@
+package fixBean;
+
+
+import quickfix.field.HeartBtInt;
+
+public class BaseFixBean {
+
+    protected String applicationID = "client";
+    protected String fileStorePath = "storage/messages/";
+    protected String fileLogPath = "outgoing";
+    protected String connectionType = "initiator";
+    protected long reconnectInterval = 60;
+//    protected String startTime = "00:01:00 Europe/Bucharest";
+//    protected String endTime = "23:59:00 Europe/Bucharest";
+    protected HeartBtInt heartBtInt = new HeartBtInt(30);
+    protected String useDataDictionary = "Y";
+    //    private DataDictionary dataDictionary = new DataDictionary(FIX42.xml);
+    protected String mmValidateUserDefinedFields = "N";
+    protected String validateIncomingMessage = "N";
+    protected String refreshOnLogon = "Y";
+    protected String NonStopSession = "Y";
+
+    public BaseFixBean() {
+    }
+
+    public BaseFixBean(String applicationID, String fileStorePath, String fileLogPath, String connectionType, long reconnectInterval, HeartBtInt heartBtInt, String useDataDictionary, String mmValidateUserDefinedFields, String validateIncomingMessage, String refreshOnLogon, String nonStopSession) {
```

Comment:
I think since have a getter/setter for each field, we can drop this constructor. We can also drop setter's if wouldn't need this bean to be mutable (Jackson would still be able load it from JSON)